schema.json

```diff
@@ -0,0 +1,392 @@
+[
```

Comment:
Column descriptions would be useful for users, but can be added later.

Reply:
While adding them, realized after few that it would be far better referring  to original documentation in protobufs. Forgot to add the link in docs. Added now.